### ISSUE TITLE
Bug: In private docs site, all URLs redirect to homepage

### ISSUE THREAD

**Issue by rhetoric101** [newrelic/docs-website#2611]
### Description

The new private docs website always seems to redirect specific URLs to the homepage. 

### Steps to reproduce

1. Go to `https://pr-8.d39l2dpntqxznx.amplifyapp.com/docs/integrations/kubernetes-integration/installation/install-fargate-integration/`
2. Note that you are redirected to this URL: `https://pr-8.d39l2dpntqxznx.amplifyapp.com/`

### Expected behavior
 
When I enter this URL, I expect to see the page at this location--not the homepage: `https://pr-8.d39l2dpntqxznx.amplifyapp.com/docs/integrations/kubernetes-integration/installation/install-fargate-integration/`

**Comment by jpvajda** [comment]
I don't believe this site supports redirects  also changing it to the correct repo.  @roadlittledawn any thoughts on this?
